### DISCUSSION TITLE
build project error

### DISCUSSION THREAD

**Discussion by tuoeg** [rabbitmq/rabbitmq-server#6068] [Other]
i build the project error according to official website，whether can provide shell command on centos for me to build,please
![image](https://user-images.githubusercontent.com/49616374/194809503-eedeb2af-a4cb-4455-a929-138c6fa9e900.png)

**Comment by michaelklishin** [comment]
Please never use GitHub issues for questions in repositories that have discussions enabled.

I can build a release using [the source tarball uploaded with other artifacts](https://github.com/rabbitmq/rabbitmq-server/releases/download/v3.11.0/rabbitmq-server-3.11.0.tar.xz) with just

``` sh
gmake
```

just fine. Either you have downloaded the source file provided by GitHub (which is not what the build system expects) or your environment is missing some prerequisites. Consider providing more details as to what exactly were the steps performed.

I'd also like to know why you need to build RabbitMQ from source while there are all kinds of pre-built packages provided.

**Reply by tuoeg** [reply]
I want to modify some rabbit-mqtt plugin function about define queueName when subscript

**Reply by tuoeg** [reply]
i have installed erlang and elixir

**Reply by tuoeg** [reply]
when use gmake to build, error occur
![image](https://user-images.githubusercontent.com/49616374/194815602-c137dc50-1a65-4953-8771-df5861c555fe.png)
